src/EditorFeatures/Core/AddImportOnPaste/AddImportOnPasteOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.AddImportOnPaste
+{
+    internal static class AddImportOnPasteOptions
```

Comment:
>AddImportOnPasteOptions [](http://example.com/codeflow?start=26&length=23)

I'd rename these new types that define the options to `XyzOptionStorage`. 

We use `XyzOptions` to store the actual values of the options when we have groups of options related to a feature.

You can find that the -Storage name is already used this way for many existing options.